orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -187,7 +187,7 @@
    <service>
         <role>LIVYSERVER</role>
         {% for hostloc in salt['pillar.get']('gateway:location')['LIVY_SERVER'] -%}
-        <url>http://{{ hostloc }}:{{ ports['LIVYSERVER'] }}</url>
+        <url>http://{{ hostloc }}:{{ ports['LIVYSERVER1'] }}</url>
```

Comment:
Is it LIVYSERVER1? or it is just a typo?

Reply:
it doesn't really matter as non of these work at the moment

Reply:
we're going to fix the URLs later, but it's blocked on the kerberos issue

Reply:
okay, then I don't understand what was the reason to change it top LIVYSERVER1 :)

Reply:
It's just a fix as it's messed up on master and the topology file cannot be generated. @pdarvasi made some changes on the exposed file, but it was not in sync with the sls file.